riff-raff.yaml

```diff
@@ -4,4 +4,14 @@ deployments:
   prism:
     type: autoscaling
     parameters:
-      bucket: deploy-tools-dist
\ No newline at end of file
+      bucket: deploy-tools-dist
+    dependencies:
+      - update-ami
+  update-ami:
+    type: ami-cloudformation-parameter
+    app: prism
+    parameters:
+      amiParameter: MachineImageAMI
+      amiTags:
+        Recipe: xenial-java8-deploy-infrastructure
+        AmigoStage: PROD
```

Comment:
@sihil Isn't the `AmigoStage` tag kinda plays the same role of making sure it has been built by amigo?